docs/openapi_generator/strong_typing/schema.py

```diff
@@ -502,11 +513,22 @@ def type_to_schema(self, data_type: TypeLike, force_expand: bool = False) -> Sch
             else:
                 output_name = property_name
 
+            # check if there's additional field annotation
+            json_schema_extra = None
+            if "model_fields" in members:
+                f = members["model_fields"]
```

Comment:
nit: you could combine this conditional into the one above which assigns all `defaults` (assign all `extras` there) and then you can simply do a simple `.get()` here

Reply:
done